server/app.ts

```diff
@@ -242,18 +247,19 @@ helpersInitialized.then(
     app.use(middleware_responseTime_start);
 
     app.use(redirectIfNotHttps);
-    app.use(express.cookieParser());
-    app.use(express.bodyParser());
+    app.use(cookieParser());
+    app.use(bodyParser.urlencoded({ extended: true }));
```

Comment:
This is now needed for accepting form-data calls that we use in `PUT /api/v3/comments` etc